tulip/transys/transys.py

```diff
@@ -709,90 +925,122 @@ def line_labeled_with(L, m=0):
     from_states = range(m, m + n - 1)
     to_states = range(m + 1, m + n)
     trans = zip(from_states, to_states)
-    ts = tuple2fts(S, S0, AP, L, Act, trans, prepend_str='s')
+    ts = tuple2fts(
+        S, S0, AP,
+        L, Act, trans,
+        prepend_str='s')
     return ts
 
 
-def cycle_labeled_with(L):
+def cycle_labeled_with(
+        L:
+            _abc.Iterable
+        ) -> FTS:
     """Return cycle FTS with given labeling.
 
-    The resulting system will be a cycle::
-        s0-> s1-> ... -> sN -> s0
-    where: N = C{len(L) -1}.
+    The resulting system will be a cycle:
 
-    See Also
-    ========
-    L{line_labeled_with}
+    ```
+    s0 -> s1 -> ... -> sN -> s0
+    ```
 
-    @param L: state labeling
-    @type L: iterable of state labels, e.g., [{'p', 'q'}, ...]
-        Single strings are identified with singleton Atomic Propositions,
-        so [..., 'p',...] and [...,{'p'},...] are equivalent.
+    where: `N = len(L) - 1`.
 
-    @return: L{FTS} with:
-        - states ['s0', ..., 'sN'], where N = len(L) -1
-        - state labels defined by L, so s0 is labeled with L[0], etc.
+    See Also
+    ========
+    `line_labeled_with`
+
+    @param L:
+        state labeling
+        iterable of state labels, e.g.,
+        `[{'p', 'q'}, ...]`
+        Single strings are identified with
+        singleton Atomic Propositions,
+        so `[..., 'p',...]` and `[...,{'p'},...]`
+        are equivalent.
+    @return:
+        `FTS` with:
+        - states `['s0', ..., 'sN']`,
+          where `N = len(L) - 1`
+        - state labels defined by `L`,
+          so `s0` is labeled with `L[0]`, etc.
         - transitions forming a cycle:
-            - s_{i} ---> s_{i+1}, for: 0 <= i < N
-            - s_N ---> s_0
+          - `s_{i} ---> s_{i+1}`,
+            for: `0 <= i < N`
+          - `s_N ---> s_0`
     """
     ts = line_labeled_with(L)
-    last_state = 's' + str(len(L) - 1)
+    last_state = f's{len(L) - 1}'
     ts.transitions.add(last_state, 's0')
-    # trans += [(n-1, 0)] # close cycle
+    # trans += [(n-1, 0)]
+    #     # close cycle
     return ts
 
 
-def add_initial_states(ts, ap_labels):
-    """Make initial any state of ts labeled with any label in ap_labels.
+def add_initial_states(
+        ts:
+            FiniteTransitionSystem,
+        ap_labels:
+            _abc.Iterable
+        ) -> None:
+    """Make initial states that match labeling.
+
+    Make initial any state of `ts` labeled with
+    any label in `ap_labels`.
 
-    For example if isinstance(ofts, FTS):
+    For example if `isinstance(ofts, FTS)`:
 
-      >>> from tulip.transys.transys import add_initial_states
-      >>> initial_labels = [{'home'}]
-      >>> add_initial_states(ofts, initial_labels)
+    ```python
+    import tulip.transys.transys as _trs
```

Comment:
From a user's perspective (i.e., the perspective of a user who reads this docstring), the name prefix `_` is not motivated, and it is easier to use `trs`:
```python
import tulip.transys.transys as trs
initial_labels = [{'home'}]
trs.add_initial_states(ofts, initial_labels)
```